internal/cloudfront/origin.go

```diff
@@ -33,6 +43,10 @@ type Origin struct {
 	Behaviors []Behavior
 	// ResponseTimeout is how long CloudFront will wait for a response from the Origin in seconds
 	ResponseTimeout int64
+	// Type is this Origin's type (s3 bucket, public, etc)
+	Type string
```

Comment:
From https://github.com/gympass/cdn-origin-controller/pull/82#discussion_r1214359839 @caiofralmeida 

> Maybe make more sense to be Access instead of Type. wdyt?